Makefile

```diff
@@ -27,6 +30,7 @@ LOCAL_BUILD_DEP:=set-up-local-build
 EXTRA_DOCKER_ARGS+=-v $(CURDIR)/../libcalico-go:/go/src/github.com/projectcalico/libcalico-go:rw
 $(LOCAL_BUILD_DEP):
 	$(DOCKER_RUN) $(CALICO_BUILD) go mod edit -replace=github.com/projectcalico/libcalico-go=../libcalico-go
+	$(DOCKER_RUN) $(CALICO_BUILD) go mod vendor
```

Comment:
I'm wary of `go mod vendor`, having a stale vendor directory around can lead to build breakage later IME; let's chat about it; my understanding was that replacing should be enough.